### DISCUSSION TITLE
web 在ios为什么加载几次就打不开了

### DISCUSSION THREAD

**Discussion by yangchongduo** [Tencent/libpag#1808] [Q&A] [answered]
web 在ios为什么加载几次就打不开了

**Comment by zhalice2011** [comment, marked as answer]
您好, 可以提供更加详细的信息吗, 机型环境(Safari浏览器 微信浏览器), 复现demo, PAG文件等

**Reply by yangchongduo** [reply]
你自己试试试不出来吗 ？不止我一个人反馈这个问题

**Reply by kevingpqi123** [reply]
pag 目前对接了了 1500 + 的业务，由于人力有限，我们没有时间为大家构造复现 demo，为了提高对接效率，请提供相关复现信息

**Reply by Mr-Zhaozhi** [reply]
同样遇到了播放几次后偶现出现PAGFile.LoadFromPath() Invalid pag file : /storage/emulated/0/Android/data/com.tal
之后再也播放不了了，重新配文件还能继续播
